### PR TITLE
Feature/model builder tests

### PR DESCRIPTION
**Pull request by marcghelas**

### CODE REVIEW

File: judokit-android/src/test/java/com/judokit/android/model/ReferenceTest.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertDoesNotThrow
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+@DisplayName("Test Reference builder")
+internal class ReferenceTest {
+    private val sut = Reference.Builder()
```
Comment (stpopa) [inline judokit-android/src/test/java/com/judokit/android/model/ReferenceTest.kt:11]: add a space before `private val sut = Reference.Builder() `

File: judokit-android/src/test/java/com/judokit/android/model/AmountTest.kt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    @DisplayName("Given all required fields are valid, then should not throw exception")
+    @Test
+    fun doesNotThrowExceptionOnAllFieldsValid() {
+        assertDoesNotThrow { sut.build() }
+    }
+
+    @DisplayName("Given formatted is called, then should return formatted amount")
+    @Test
+    fun returnFormattedAmountOnFormattedCalled() {
+        assertEquals("GBP1.00", sut.setCurrency(Currency.GBP).build().formatted)
```
Comment (stpopa) [inline judokit-android/src/test/java/com/judokit/android/model/AmountTest.kt:47]: should it be `£1.00`?

File: judokit-android/src/test/java/com/judokit/android/api/model/request/GooglePayWalletTest.kt [hunk 3]
```diff
@@ -8,6 +9,9 @@ import org.junit.jupiter.api.assertThrows
 internal class GooglePayWalletTest {
 
     private val wallet = GooglePayWallet.Builder()
+        .setCardDetails("cardDetails")
+        .setCardNetwork("network")
+        .setToken("token")
 
     @Test
```
Comment (stpopa) [inline judokit-android/src/test/java/com/judokit/android/api/model/request/GooglePayWalletTest.kt:17]: extra space

File: judokit-android/src/test/java/com/judokit/android/model/AmountTest.kt (hunk 2 above)
Comment (marcghelas) [reply judokit-android/src/test/java/com/judokit/android/model/AmountTest.kt:47]: it is the pound symbol on device, but in tests it returns GBP for some reason

**Pull request merged**